docs/_layouts/global.html

```diff
@@ -87,8 +89,8 @@
                                         </a>
                                     </li>
                                 {% else %}
-                                    <!-- trim the leading slash to make the path relative to preserve version links -->
-                                    <li><a href="{{ node.url | remove-first: '/' }}">
+                                    <!-- Use the prefix .. to change the absulote url to relative url -->
+                                    <li><a href="..{{ node.url }}">
```

Comment:
Yes. Since the documentation site isn't on the root path of the website, absolute path will cause error. But this approach means all page files must be in en/ (or cn/) directory exactly and no subdirectory could be created to contain pages. I'm also wondering is there a better approach for this case.